tests/helper/helper_dev.go

```diff
@@ -105,10 +127,23 @@ func (o DevSession) Stop() {
 	o.session.Interrupt()
 }
 
+func (o DevSession) WaitSync() ([]byte, []byte) {
+	WaitForOutputToContain("Pushing files...", 180, 10, o.session)
+	WaitForOutputToContain("Waiting for something to change", 180, 10, o.session)
+	out := o.session.Out.Contents()
+	err := o.session.Err.Contents()
+	o.session.Out.Clear()
+	o.session.Err.Clear()
+	return out, err
+}
+
+func (o DevSession) WaitForOutputToContain(substring string, timeoutInSeconds int, intervalInSeconds int) {
```

Comment:
What's the purpose of this function? I don't see this getting used anywhere in the tests:
```sh
$ grep -r "WaitForOutputToContain" tests/integration
tests/integration/devfile/cmd_dev_test.go:				helper.WaitForOutputToContain("Error occurred on Push", 180, 10, session)
tests/integration/devfile/cmd_dev_test.go:			helper.WaitForOutputToContain("Waiting for something to change", 180, 10, session)
tests/integration/devfile/cmd_dev_test.go:			helper.WaitForOutputToContain("server.js", 180, 10, session)
tests/integration/devfile/cmd_dev_test.go:				helper.WaitForOutputToContain("Waiting for something to change", 180, 10, session)
tests/integration/devfile/cmd_dev_test.go:					helper.WaitForOutputToContain("Waiting for something to change", 180, 10, session)
tests/integration/devfile/cmd_delete_test.go:				helper.WaitForOutputToContain("Press Ctrl+c to exit", 180, 10, session)
tests/integration/devfile/cmd_delete_test.go:			helper.WaitForOutputToContain("Press Ctrl+c to exit", 180, 10, session)
```

Reply:
Not used anymore, I remove it